setup.cfg

```diff
@@ -13,6 +13,7 @@ parentdir_prefix = pandas-
 
 [flake8]
 ignore = E731
+max-line-length = 79
```

Comment:
Some people will have local defaults that are different. This will override those